packages/docusaurus/lib/webpack/plugins/ChunkManifestPlugin.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
```

Comment:
This is the main logic to build chunk names -> chunk assets mapping. This plugin is not available anywhere ! Not a fork !

Reply:
Should this be a package on its own?

Reply:
This is more fit to use for docusaurus itself actually. Kinda hard to maintain it as a package and had to make it general